prez-components/src/types.ts

```diff
@@ -1,40 +1,51 @@
-import { ListItem, PrezTerm, PrezLiteral, PrezNode } from "prez-lib";
-
-// having trouble accessing the prez-lib from other locations.
-export type UI_PrezLiteral = PrezLiteral;
-
-export interface PrezUITermProps {
-    color?: "primary" | "secondary";
-    term: PrezTerm;
-}
-
-export interface PrezUILiteralProps {
-    color?: "primary" | "secondary";
-    term: PrezLiteral;
-}
-
-export interface PrezUINodeProps {
-    color?: "primary" | "secondary";
-    size?: "normal" | "small";
-    term: PrezNode;
-}
-
-export interface ButtonProps {
-    color?: "primary" | "secondary";
-    size?: "sm" | "lg";
```

Comment:
Thoughts on separating the data object from the UI object as previously done with the term property vs extending the data objects? I found it harder to distinguish in some areas which properties were part of the data and which were specific to the UI properties.

Reply:
See above